cairo/src/font/scaled_font.rs

```diff
@@ -180,9 +163,9 @@ impl ScaledFont {
                 y,
                 text.as_ptr(),
                 text_length,
-                &mut glyphs_ptr,
+                &mut glyphs_ptr as *mut _ as *mut _,
                 &mut glyph_count,
-                &mut clusters_ptr,
+                &mut clusters_ptr as *mut _ as *mut _,
```

Comment:
Add some more explicit casts here, but also only one cast should be needed

Reply:
Also explicit casts elsewhere

Reply:
No because we first need to convert from `&mut` to `*mut` and then to the correct pointer type. However I'll update for explicit cast.